usaspending_api/data/authoritative_agency_list.csv

```diff
@@ -1283,7 +1283,7 @@ CGAC AGENCY CODE,FPDS DEPARTMENT ID,AGENCY NAME,AGENCY ABBREVIATION,REGISTERED I
 097,9700,Department of Defense,DOD,YES,YES,NIST (2008),21CB,U.S. Army Criminal Investigation Command,,,,DE00,Department of Defense,,,,,,FALSE,,,
 097,9700,Department of Defense,DOD,YES,YES,NIST (2008),21CS,Immediate Office of the Chief of Staff of the Army,,,,DE00,Department of Defense,,,,,,FALSE,,,
 097,9700,Department of Defense,DOD,YES,YES,NIST (2008),21CZ,U.S. Army Information Systems Command,,,,DE00,Department of Defense,,,,,,FALSE,,,
-097,9700,Department of Defense,DOD,YES,YES,NIST (2008),21E1,Immediate Office � Commander-In-Chief - U.S. Army Europe and 7th Army,,,,DE00,Department of Defense,,,,,,FALSE,,,
+097,9700,Department of Defense,DOD,YES,YES,NIST (2008),21E1,Immediate Office � Commander-In-Chief - U.S. Army Europe and 7th Army,,,,DE00,Department of Defense,,,,,,FALSE,,,

```

Comment:
should we fix this special character? In the broker agency code list the character is `–`